Source/DeviceUtil.m

```diff
@@ -11,9 +11,11 @@
 #import "DeviceUtil+Constant.h"
 #include <sys/sysctl.h>
 
-@implementation DeviceUtil {
-  NSDictionary *deviceList;
-}
+@interface DeviceUtil()
+@property (nonatomic, strong) NSDictionary *deviceList;
+@end
+
+@implementation DeviceUtil
```

Comment:
Isn't the both code same? I'm out of touch with objc can you please explain the problem and what prevented it